src/estimate_rbc_1_joint.jl

```diff
@@ -0,0 +1,104 @@
+    m = FirstOrderPerturbationModel(rbc_1)
+    turing_model = rbc_joint(
+        z, m, d.p_f, d.alpha_prior, d.beta_prior, d.rho_prior, allocate_cache(m)
+    )
+
+    # Sampler
+    name = "rbc-joint-s$(d.num_samples)-seed$(d.seed)"
+    include_vars = ["α", "β_draw", "ρ"]  # variables to log
+    callback = TensorBoardCallback(d.results_path; name, include=include_vars)
+    num_adapts = convert(Int64, floor(d.num_samples * d.adapts_burnin_prop))
+
+    Random.seed!(d.seed)
+    @info "Generating $(d.num_samples) samples with $(num_adapts) adapts"
+    alg = NUTS(num_adapts, d.target_acceptance_rate)
+
+    chain = sample(
+        turing_model,
+        NUTS(num_adapts, d.target_acceptance_rate; d.max_depth),
+        d.num_samples;
+        init_params=[d.p; ϵ0],
+        progress=true,
+        save_state=true,
+        callback,
+    )
+
+    # Calculate and save results into the logdir
+    calculate_experiment_results(chain, callback.logger, include_vars, d.full_results)
+    
+    # Store parameters in log directory
+    parameter_save_path = joinpath(callback.logger.logdir, "parameters.json")
+
+    @info "Storing Parameters at $(parameter_save_path) "
+    open(parameter_save_path, "w") do f
+        write(f, JSON.json(d))
+    end
+end
+
+function parse_commandline_rbc_1_joint(args)
+    s = ArgParseSettings(; fromfile_prefix_chars=['@'])
+
+    # See the appropriate _defaults.txt file for the default vvalues.
+    @add_arg_table! s begin
+        "--data_path"
+        help = "relative path to data from the root of the package"
+        arg_type = String
+        "--p"
+        help = "Initialization of parameters"
+        arg_type = Vector{Float64}
+        "--p_f"
+        help = "Value of fixed parameters"
+        arg_type = Vector{Float64}
+        "--alpha_prior"
+        help = "Parameters for the prior"
+        arg_type = Vector{Float64}
+        "--beta_prior"
+        help = "Parameters for the prior"
+        arg_type = Vector{Float64}
+        "--rho_prior"
+        help = "Parameters for the prior"
+        arg_type = Vector{Float64}
+        "--num_samples"
+        help = "samples to draw in chain"
+        arg_type = Int64
+        "--adapts_burnin_prop"
+        help = "Proportion of Adaptations burned in"
+        arg_type = Float64
+        "--target_acceptance_rate"
+        help = "Target acceptance rate for dual averaging for NUTS"
+        arg_type = Float64
+        "--max_depth"
+        help = "Maximum depth for NUTS"
+        arg_type = Int64
+        "--seed"
+        help = "Random number seed"
+        arg_type = Int64
+        "--results_path"
+        arg_type = String
+        help = "Location to store results and logs"
+        "--full_results"
+        arg_type = Bool
+        help = "Save the complete set of figures and results for the chain"
+    end
+
+    args_with_default = vcat("@$(pkgdir(HMCExamples))/src/rbc_1_kalman_defaults.txt", args)
```

Comment:
Could you copy the `rbc_1_kalman_defaults.txt` file and make a `rbc_1_joint_defaults.txt` file? I don't think anything should change meaningfully. 

Might be worth it later on to just remove these entirely, but for right now let's stick to the format Jesse's got.

Reply:
Ok, sure, will do this

Reply:
Just added the commit